pipeline/summarize_results.py

```diff
@@ -31,6 +32,52 @@
 from nemo_skills.evaluation.metrics import MathEval
 from nemo_skills.utils import setup_logging
 
+LOG = logging.getLogger(__name__)
+
+
+def process_batch_results(prediction_jsonl_files):
```

Comment:
Can you please reuse this logic in here https://github.com/Kipok/NeMo-Skills/blob/main/nemo_skills/evaluation/metrics.py#L346 and in here https://github.com/Kipok/NeMo-Skills/blob/main/nemo_skills/evaluation/metrics.py#L54

Reply:
Do you want me to rewrite `def setup(self, prediction_jsonl_files):` the way I did `def process_batch_results(prediction_jsonl_files):`, or vice versa?